tests/test_solvers/test_harbor_terminus_patches.py

```
@@ -0,0 +1,463 @@
+        assert result == 50
+        assert "shrank below the API anchor" not in caplog.text
+
+
+# ── anchored estimate + custom tokenizer integration ─────────────────────
+
+
+class TestAnchoredEstimateUsesCustomTokenizer:
+    """The anchored estimate routes its token_counter calls through the custom tokenizer."""
+
+    @pytest.fixture(autouse=True)
+    def _reset_counter_patch(self, monkeypatch):
```

Ruff (0.15.18)

[warning] Missing return type annotation for private function `_reset_counter_patch`

Add return type annotation: `None`

(ANN202)